<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeamspeakBadgesViewerClass</class>
 <widget class="QMainWindow" name="TeamspeakBadgesViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Teamspeak Badges Parser</string>
  </property>
  <property name="windowIcon">
   <iconset resource="TeamspeakBadgesViewer.qrc">
    <normaloff>:/icon/resource/icon.png</normaloff>:/icon/resource/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(26, 35, 53);
color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget QScrollBar:vertical {              
border: 0px solid white;
width:12px;
background: white;
}

#centralwidget QScrollBar::handle:vertical {
background: rgb(32, 47, 130);
min-height: 1px;
border: 2px solid white;
border-radius: 5px;
}

#centralwidget QLabel a {
text-decoration: none;
}</string>
   </property>
   <widget class="QTableWidget" name="badgesTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>281</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget{ 
background-color: rgb(43, 51, 70);
border: 1px solid #1c82cc;
border-radius: 5px;
}

/*#centralwidget QScrollBar:vertical {              
border: 0px solid white;
width:12px;
background: white;
}

#centralwidget QScrollBar::handle:vertical {
background: rgb(32, 47, 130);
min-height: 1px;
border: 2px solid white;
border-radius: 5px;
}*/
/*QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {background: none;}*/

</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>300</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>300</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <row/>
    <column/>
   </widget>
   <widget class="QPushButton" name="clearCacheBtn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>420</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 51, 70);
border: 1px solid #1c82cc;
border-radius: 5px;
color: white;</string>
    </property>
    <property name="text">
     <string>Clear Cache</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>291</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 51, 70);
border-radius: 5px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="iconaBadge">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #1c82cc;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="nomeBadge">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #1c82cc;
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QLabel" name="descBadge">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #1c82cc;
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>291</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 51, 70);
border-radius: 5px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="uuid_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #1c82cc;
border-radius: 5px;
color: white;</string>
     </property>
     <property name="text">
      <string>UUID</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="uuidText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel::hover{
 border-bottom: 1px solid #1c82cc;
 border-radius: 5px;
}
QLabel::!hover{
 border-bottom: 1px solid #48546c;
 border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="img_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #1c82cc;
border-radius: 5px;
color: white;</string>
     </property>
     <property name="text">
      <string>IMG links</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="svgUrl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel::hover{
 border-bottom: 1px solid #1c82cc;
 border-radius: 5px;
}
QLabel::!hover{
 border-bottom: 1px solid #48546c;
 border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="detailsSvgUrl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel::hover{
 border-bottom: 1px solid #1c82cc;
 border-radius: 5px;
}
QLabel::!hover{
 border-bottom: 1px solid #48546c;
 border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="pngUrl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel::hover{
 border-bottom: 1px solid #1c82cc;
 border-radius: 5px;
}
QLabel::!hover{
 border-bottom: 1px solid #48546c;
 border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="detailsPngUrl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel::hover{
 border-bottom: 1px solid #1c82cc;
 border-radius: 5px;
}
QLabel::!hover{
 border-bottom: 1px solid #48546c;
 border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="statusMessage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>420</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Badges will update after 7 days that the file was created</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
   </widget>
   <addaction name="menuInfo"/>
  </widget>
 </widget>
 <resources>
  <include location="TeamspeakBadgesViewer.qrc"/>
 </resources>
 <connections/>
</ui>
